config/environments/production.rb

```diff
@@ -59,7 +59,19 @@
 
   # Ignore bad email addresses and do not raise email delivery errors.
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
-  # config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :smtp
+  host = 'protected-lowlands-2991.herokuapp.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.